### ISSUE TITLE
About the use of outliers

### ISSUE THREAD

**Issue by haoni0812** [FlyingRoastDuck/MetaCam_DSCE#5]
Hi, thanks for sharing your work. I noticed that the outliers seem not to be used in the code (as shown below). However,  according to the description in the paper, outliers should be assigned to the nearest pseudo category. I would like to know where this operation is carried out.

in em.py:   nums.index_add_(0, labels[labels != -1], torch.ones(labels[labels != -1].shape[0], 1).float().cuda())

**Comment by FlyingRoastDuck** [comment]
Hi, Sorry for the late reply. You can find the ``outlier merging" operation here:

https://github.com/FlyingRoastDuck/MetaCam_DSCE/blob/master/reid/utils/__init__.py#L43

if you have further questions, please contact me through email. I cannot reply to you timely with github issues.
